packages/core/src/transport/transport.ts

```diff
@@ -29,9 +64,32 @@ export class HttpRequest {
         reportBeaconError(e)
```

Comment:
It may be interesting to use the same "transport error context" for beacon errors too.

Reply:
Good idea. Let's see how this works out first?